src/static/kanban.js

```diff
@@ -154,7 +178,7 @@ window.app = new Vue({
     init: function () {
       this.refresh_columns();
       this.refresh_cards();
-      this.get_all_users();
+      // this.get_all_users();
```

Comment:
@sweiyang , any reason for this to be commented out? I going to put this back for me to get all users for the project. So that the task assigned by user email

Reply:
This literally gets all users. It should be replaces with get_all_users_by_project_id

Reply:
ahh misunderstanding of the function name.  Inside the function already calling /user/get_user_by_project_id. I will refactor the function name. 

![image](https://user-images.githubusercontent.com/10574652/236617592-440497a6-cb37-4579-9989-62fec3b38524.png)